package.json

```diff
@@ -107,7 +107,7 @@
     "api-watch": "nodemon -e \".ts\" --exec npm run babel-node-api",
     "babel-node-api": "babel-node -r dotenv/config -x \".ts\" ./src/api/api.ts",
     "build-api": "npx tsc --build ./tsconfig-api.json",
-    "test-api": "npx jest --testTimeout=5000 --forceExit --runInBand --config=./src/api/jest.config.ts --runInBand",
+    "test-api": "npx jest --testTimeout=5000 --forceExit --runInBand --config=./src/api/jest.config.ts",
```

Comment:
Are we removing this because we already have the same flag earlier in the command?